src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -366,9 +411,33 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                     scene.pointerY = this._screenCoordinatesRef.y;
                 }
             }
-            controllerData.pick = this._scene.pickWithRay(controllerData.tmpRay, this._scene.pointerMovePredicate || this.raySelectionPredicate);
 
-            const pick = controllerData.pick;
+            let hover = this._pickWithMesh(controllerData.hoverMesh, false);
+            let nearHover = hover && hover.pickedPoint && hover.hit;
+            if(nearHover)

```

Comment:
If we're in hover range, we should probably commit fully to the near interaction model. The way its laid out here, we can send hover info, but still do a far interaction, which creates interactions on potentially two different objects at the same time (not supported by the BJS pointer system/controls)